bedrock/exp/templates/exp/firefox/mobile.html

```diff
@@ -0,0 +1,217 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+{% from "macros-protocol.html" import feature_card, hero, call_out_compact with context %}
+
+{% extends "exp/base/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('exp-firefox-mobile') }}
+{% endblock %}
+
+{% block canonical_urls %}
+  {# Experimental pages should set canonical to the original page URL. #}
+  <link rel="canonical" href="{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/mobile">
```

Comment:
Nit add trailing slash?
```suggestion
  <link rel="canonical" href="{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/mobile/">
```